lib/js/atoms/SurveyQuestion/SurveyQuestion.domain.ts

```diff
@@ -0,0 +1,7 @@
+export interface ScaleOption {
```

Comment:
It will be imported as `import { ScaleOption } from 'design-system'`, right? I don't know what the best practice for interface names is, but at least I'd add a prefix `SurveyQuestionScaleOption`

Reply:
[`02a7207` (#71)](https://github.com/bethinkpl/design-system/pull/71/commits/02a7207da18a00ad30f0a20923bd9ed20602daba)